components/src/core/drawer/drawer-body/nav-item/drawer-nav-item.component.scss

```diff
@@ -55,23 +52,24 @@ pxb-drawer-nav-item {
         padding: 0;
     }
 
-    .pxb-info-list-item-active.round:after,
-    .pxb-info-list-item-active.square:after {
-        content: '';
+    .pxb-drawer-nav-item-active-highlight {
+        display: flex;
         z-index: 0;
         position: absolute;
         height: 100%;
         left: 0;
         top: 0;
         opacity: 0.9;
-    }
-    .pxb-info-list-item-active.round:after {
-        width: calc(100% - 8px);
-        border-radius: 0 24px 24px 0;
-    }
-    .pxb-info-list-item-active.square:after {
-        border-radius: 0;
-        width: 100%;
+        min-width: 120px;
```

Comment:
Can we drop the min-width here? If user sets width of the drawer to less than this, the round part get clipped.

Reply:
The only reason why this is here to band-aid the border radius that appears when selecting/opening an item in the persistent variant.

This is the issue that I was seeing:  When opening the drawer, the rounded border of the selected item arrives too early. 
![circle](https://user-images.githubusercontent.com/6538289/91856311-67136400-ec34-11ea-947e-7d076afb912b.gif)


This is what a 120px drawer looks like: 
![image](https://user-images.githubusercontent.com/6538289/91855613-6af2b680-ec33-11ea-9b17-807d850f2183.png)

This fix just tries to mask when the rounded corners show up; I don't think it's a very elegant fix.  Any idea on how to tweak it to make it work with any width?

Reply:
Another idea I had would be to wait until the drawer is opened (animation over) before applying the `round` class onto the highlight area.

Reply:
You might be able to animate/transition the border radius too. That might help smooth it out.